docs/tutorials/third-party-result-schema-SCC.md

```diff
@@ -0,0 +1,255 @@
+# Schema of Assessment Results for Interchange with SCC
+
+This document depicts the structure and guidelines for creating OSCAL result object which would be generated by various transformers for different tools such as tanium to OSCAL and OSCO to OSCAL. PVPs such as toolchain that directly generate OSCAL result for sending to SCC Exchange API should follow this structure and guidelines.
+
+The overall object will be a *results* element of OSCAL Assessment Result schema warpped in an outer json object as shown below.
```

Comment:
There are a number of definitions here which need to be explained to a user who is not familiar with IBM tooling (SCC, Exchange API, OSCO, PVPs). We need to make this more approachable to external users.